texar/utils/utils.py

```diff
@@ -103,7 +106,7 @@ def no_map(container_type: Type[T], *args, **kwargs) -> T:
         args: Arguments to the constructor.
         kwargs: Keyword arguments to the constructor
 
-    Returns:
+    :returns:
```

Comment:
There's no need to change `Returns:` to `:returns:` if the return value description does not contain colons (`:`).